makefile_include.mk

```diff
@@ -13,9 +13,18 @@ ifndef CROSS_COMPILE
   CROSS_COMPILE:=
 endif
 
-ifeq ($(CC),cc)
+ifeq ($(origin CC),default)
```

Comment:
Before merging this... I just realized that the original check for `CC` was already wrong, this path should only be taken if `CROSS_COMPILE` is defined... or not? :)

Reply:
\*ping\*?

Reply:
Sorry, been busy with other stuff. I'll revisit this.